libs/base/ui/overlays/dropdown/dropdown.styles.ts

```diff
@@ -23,7 +23,7 @@ export const dropdownBaseStyles = css`
       var(--oryx-popover-maxwidth)
     );
 
-    z-index: 100;
+    /* z-index: 100; */
```

Comment:
what's the resason for this one?

Reply:
This has been a mistake from the beginning. I've however wrapped it inside a featureVersion to keep BC.